SupersonicSound/Wrapper/Util.cs

```diff
@@ -207,5 +208,14 @@ public static Guid FromFmod(this GUID guid)
                 guid.Data4[7]
             );
         }
+
+        public static bool IsLinux
+        {
+            get
+            {
+                int p = (int)Environment.OSVersion.Platform;
```

Comment:
Just took it from here: http://stackoverflow.com/questions/5116977/how-to-check-the-os-version-at-runtime-e-g-windows-or-linux-without-using-a-con, based on the Mono FAQ.